From 6f994b957c0749478843c021d6bb2be0917a7c2f Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 15 Apr 2013 08:54:42 +0800
Subject: [PATCH 680/691] ENGR00258651 mx6: update thermal equation for
 i.MX6sl

New thermal equation is working for all i.MX6 series
SOCs, so update it for i.MX6SL as well.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 cpu/arm_cortexa8/mx6/generic.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cpu/arm_cortexa8/mx6/generic.c b/cpu/arm_cortexa8/mx6/generic.c
index 253b5e0..9aea30d 100644
--- a/cpu/arm_cortexa8/mx6/generic.c
+++ b/cpu/arm_cortexa8/mx6/generic.c
@@ -892,11 +892,11 @@ static inline int read_cpu_temperature(void)
 	hot_temp = fuse & 0xff;
 
 	/*
-	 * Only when it is i.MX6Q and high temperature calibration
+	 * Only when high temperature calibration
 	 * data not used, we use universal equation to get thermal
 	 * sensor's ratio.
 	 */
-#if (!defined(CONFIG_MX6SL) && !defined(USE_CALIBRATION))
+#ifndef USE_CALIBRATION
 	/*
 	 * The universal equation for thermal sensor
 	 * is slope = 0.4297157 - (0.0015976 * 25C fuse),
-- 
1.7.7.4

